source/objects/lander.lua

```diff
@@ -64,7 +64,7 @@ local function parachuteIsDeployed(lander)
 -- return true if lander has a parachute and it is deployed
 
 	for _, moduleItem in pairs(lander.modules) do
-		if moduleItem.id == 5 then	-- 5 = parachute	
+		if moduleItem.id == Enum.moduleParachute then	
```

Comment:
I already said this like 100 times but I don't like the usage of enums / indexes to point to a specific table entry because it just adds extra complexity to the code, requires an additional variable to be said every time and just is not as pleasant to read as using a string or the table key directly like `Modules.thrusters`.